Makefile.js

```diff
@@ -359,6 +355,22 @@ target.changelog = function() {
 	mv('CHANGELOG.md.tmp', 'CHANGELOG.md');
 };
 
+target['update-readme-version'] = function() {
```

Comment:
Do you need to run this as a separate build target? If not, might be safer be a regular function and pass in the version to update rather than relying on git tags for that signal.